posit_workbench_login_guidance.md

```diff
@@ -0,0 +1,39 @@
+# Guidance to log in and start a session on Posit Workbench
+
+Original Author: Ciara Gribben
+
+Last Updated: December 2022
```

Comment:
This is kind of a general comment for everything we do as I know it's included in templates, but as we're using version control this information isn't required.